code/modules/response_team/ert_outfits.dm

```diff
@@ -447,6 +447,13 @@
 	r_pocket = /obj/item/nullrod/ert
 	glasses = /obj/item/clothing/glasses/sunglasses
 
+	backpack_contents = list(
+		/obj/item/clothing/shoes/magboots = 1,
```

Comment:
Red Security and Red Commander actually has their magboots in their backpack as well, so there is a precedent to it being in the backpack rather than spawning with it on their legs. I don't mind changing it though if it is a big issue, though that would also require changing Red Security and Red Commander's loadouts too.

Reply:
There isn't really a precedent, some were changed and some were not. Some of them were updated to magboots when I made changes to ERT last year. I would say its probably better to have them on the feet in this case? I just don't see the point of ERT having to mess around with shoes when they are already racing the clock in most cases.

Reply:
So should I make Red Commander and Security also get magboots on their feet too?

Reply:
I would say just leave it to the role you have it changed for now. We can just make more adjustments in another PR later. It isn't anything critical, it just is eliminating some extra bloat from the kits overall.